rust/hyper/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+http-body-util = "*"
+hyper = { version = "1.5", features = ["http1", "server"] }
+hyper-util = { version = "*", features = ["http1", "server", "tokio"] }
+tokio = { version = "1", features = ["rt", "rt-multi-thread", "net", "macros"] }
```

Comment:
is version required => https://github.com/hyperium/hyper/blob/30f2961e89eb306780d856e6e2c1ee10ffbbafd2/Cargo.toml#L26 ?

Reply:
if you mean _can we replace "1" with "*"_ - no we cannot, cargo will happily add two versions of the same crate to the dependency graph if they are _not_ semver compatible